package.json

```diff
@@ -58,25 +57,26 @@
     "babel-preset-react": "^6.11.1",
     "coveralls": "^2.11.14",
     "cz-conventional-changelog": "^2.0.0",
-    "enzyme": "^2.4.1",
+    "enzyme": "^3.0.0",
+    "enzyme-adapter-react-16": "^1.0.0",
     "eslint": "^4.1.0",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-react": "^7.0.0",
     "nodemon": "^1.11.0",
     "nyc": "^11.0.1",
-    "react-addons-test-utils": "^15.3.2",
     "react-redux": "^5.0.4",
-    "sinon": "^2.1.0"
+    "react-test-renderer": "^16.0.0",
+    "sinon": "^4.0.0"
   },
   "dependencies": {
     "babel-runtime": "^6.11.6",
-    "debug": "^2.2.0",
+    "debug": "^3.1.0",
     "invariant": "^2.2.1",
-    "react": "^15.3.2",
-    "react-dom": "^15.3.2",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
     "redux": "^3.6.0",
-    "redux-observable": "^0.14.1",
-    "rxjs": "^5.3.0"
+    "redux-observable": "^0.16.0",
+    "rxjs": "^5.0.0"
```

Comment:
Why was this downgraded?

Reply:
that was when I was still playing around with peerDep stuff, has been fixed.